packages/admin-ui/ui/src/components/molecules/tree-crumbs/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import { ProductCategory } from "@medusajs/medusa"
+
+type TreeCrumbsProps = React.HtmlHTMLAttributes<HTMLDivElement> & {
+  nodes: ProductCategory
+  currentNode: ProductCategory
+}
+
+const TreeCrumbs: React.FC<TreeCrumbsProps> = ({
+  nodes,
+  currentNode,
+  showPlaceholder = false,
+  ...props
+}) => {
+  const parentNode = nodes.find((n) => n.id === currentNode?.parent_category_id)
+
+  return (
+    <span {...props}>
+      <span className="text-grey-40">
+        {parentNode && (
+          <TreeCrumbs
+            nodes={nodes}
+            currentNode={parentNode}
+            showPlaceholder={false}
+          />
+        )}
+
+        {parentNode && <span className="mx-2">/</span>}
+
+        {currentNode.name}
+
+        {showPlaceholder && (
+          <span>
+            <span className="mx-2">/</span>
+
+            <span className="border-grey-40 rounded-[10px] border-[1px] border-dashed px-[8px] py-[4px]">
```

Comment:
suggestion: we could add something like:

https://user-images.githubusercontent.com/16856471/223826912-016a7682-2055-4f44-8262-414da1029d47.mov

But up to you :)

Reply:
Lovely, adding this in.